Makefile

```diff
@@ -47,7 +60,7 @@ ifeq ($(RUN_LONG_TESTS), yes)
 	PATH=$(BINPATH)/:$(PATH) && . system/env/bin/activate && APTLY_VERSION=$(VERSION) $(PYTHON) system/run.py --long $(TESTS)
 endif
 
-test:
+test: etcd-prepare
```

Comment:
This will re-download `etcd` everytime tests run, right? That seems quite wasteful. Maybe we can check if etcd exists and don't redownload it every time?